src/components/search/result/row/Row.js

```diff
@@ -68,6 +70,23 @@ TitlesForSearch.propTypes = {
   work: PropTypes.object,
   loading: PropTypes.bool,
 };
+
+function sortMaterialTypesByRouter(routerMaterialTypes) {
+  return (a, b) => {
+    const promoteA = routerMaterialTypes?.findIndex((mat) =>
+      isEqual(a.join(","), mat)
```

Comment:
er det pga sammensatte materialer .. dender join ting ?

Reply:
Ja, men ikke kun derfor.

Ja, fordi materialetyperne er i arrays, som oftest kun har én materialeType, men med sammensatte har den flere.

Men det er også fordi filtrene faktisk ikke bruger de rigtige materialetyper og bruger factoriet. For som filtrene er nu, kan man kun filtrere på "sammensat materiale". 

Det skal muligvis rettes senere, når filtrene får de egentlige materialeTyper, men jeg tror faktisk allerede jeg kan forudsige, at det ikke kommer til at være `,`, men derimod ` / `, så jeg retter det bare med det samme...!